client/src/redux/reducers/datafiles.reducers.js

```diff
@@ -370,12 +370,14 @@ export function files(state = initialFilesState, action) {
         }
       };
     case 'STORE_SELECTOR_REF':
-      return {
-        ...state,
-        refs: {
-          ...state.refs,
-          FileSelector: action.payload
-        }
+      if(action.payload.props.isOpen){
```

Comment:
This still causes the issue if you
1. Open select modal
2. Close select modal
3. Run app and need to push keys

Reply:
Ah you're right, had to refresh.